documentation/docs/libraries/java/how_to/_run_how_tos.md

```diff
@@ -19,7 +19,7 @@ you can run to get acquainted with the library. You can run any example with the
 command from the `examples` directory:
 
 ```bash
-.././gradlew run -Pmain=CreateAccount
+.././gradlew run -Pexample=CreateAccount
```

Comment:
the build.gradle still has
`mainClassName = project.findProperty("main").toString()`

Reply:
changed, thanks!